### ISSUE TITLE
res option in akde()

### ISSUE THREAD

**Issue by chfleming** [ctmm-initiative/ctmmweb#82]
It would be nice if users could lower the `res` argument in `akde()` if they run out of RAM when calculating home ranges. This can happen with disparate overlapping home ranges. The default is 10, but it can be reduced down to about 1 without horrendous consequence.

**Comment by xhdong-umd** [comment]
Home range will be updated automatically once switched to that page. We can put an option but user can only change it after the update finish so it will not be useful for out of RAM problem. Unless we change the default to 1 first.

**Comment by chfleming** [comment]
I would not default to 1. The RAM error will happen very quickly, because the array will fail to initialize in memory. The user will be able to adjust the `res` and proceed, right?

**Comment by xhdong-umd** [comment]
If the ram error will not crash the app then this should be fine. Is there any data or script to reproduce the error so I can test it?

**Comment by chfleming** [comment]
You can reproduce it by following the `overlap` example (https://ctmm-initiative.github.io/ctmm/reference/overlap.html), but moving one of the buffalo very far away from the other before calculating anything, like

buffalo[[2]]$x <- buffalo[[2]]$x + 10^10

**Comment by xhdong-umd** [comment]
I added code to catch the error and fall back with `res=1`. It worked in script with 2 individuals. 

In web app it still have memory error with 6 individuals. I limited selection to 2 individuals, the computation finished, console log updated normally, but the app is very slow to response. System monitor said R session used 15.7 G memory. After quite some time the app showed the plot and summary. So this will work but only within the memory limit.
